deeplinkdispatch-processor/build.gradle

```diff
@@ -13,7 +13,8 @@ dependencies {
     testCompile 'junit:junit:4.12'
     testCompile 'org.assertj:assertj-core:1.7.0'
     testCompile 'com.google.android:android:4.1.1.4'
-    testCompile 'com.google.android:support-v4:r7'
+    //testCompile 'com.google.android:support-v4:r7'
+    testCompile fileTree(dir: 'libs', include: ['android-support-v4.jar'])
```

Comment:
i've tried this but no success. Dependency is resolved on syncing the project but not at run time.
jcenter provides aar with contains jar inside libs for android.support.v4.TaskStackBuilder which can't be resolved at runtime...
This is similar to issue referred in [ Jar files in libs folder of in AAR package was not picked up](https://github.com/simpligility/android-maven-plugin/issues/484)